rollrus.go

```diff
@@ -9,20 +9,42 @@ import (
 	"github.com/stvp/roll"
 )
 
+var defaultFiredLevels = []log.Level{
+	log.ErrorLevel,
+	log.FatalLevel,
+	log.PanicLevel,
+}
+
 // Hook wrapper for the rollbar Client
 // May be used as a rollbar client itself
 type Hook struct {
 	roll.Client
+	firedLevels []log.Level
```

Comment:
Similarly to `defaultLevels` I would just call this either `levels` or `triggerOn` (I dislike "fired" for some reason).